assets/_scss/elements/_typography.scss

```diff
@@ -17,12 +17,22 @@ p {
 }
 
 a {
-  color: $color-base-link;
-  &:hover, &:active, &:visited {
-    color: $color-base-link;
+  border-bottom: 1px dotted;
+  &:hover {
+    border-bottom: 1px solid;
   }
 }
 
+.usa-link {
+  border-bottom: none;
+  font-family: $font-sans; 
+  font-weight: 700;
```

Comment:
Yea it could follow typography conventions: $light, $book, $medium, $bold, $exbold, whatever. In reality, most web fonts will only use 2 or 3. Maybe just $light, $medium, $bold will do, but you might want to talk to the designers.